server/datastore/mysql/apple_mdm_test.go

```diff
@@ -3757,18 +3756,15 @@ func TestHostDEPAssignments(t *testing.T) {
 		require.True(t, *h.DEPAssignedToFleet)
 		h, err = ds.LoadHostByDeviceAuthToken(ctx, depDeviceTok, 1*time.Hour)
 		require.NoError(t, err)
+		h, err = ds.LoadHostByDeviceAuthToken(ctx, depDeviceTok, 1*time.Hour)
+		require.NoError(t, err)
```

Comment:
Same thing, lines 3759-3760 are not needed/were already present, only line 3761 was missing.